src/js/views/emailNotificationView.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+var Marionette = require('backbone.marionette'),
+    _ = require('underscore');
+
+var EmailNotificationTemplate = require('../../templates/emailNotification.tpl');
+
+module.exports = Marionette.ItemView.extend({
+    tagName: 'p',
```

Comment:
Backbone views (Marionette.ItemView inherits from it) generates a `div` by defautlt as the container for the view. `tagName` allows you to change that to something else.